src/compiler/compile/nodes/Element.ts

```diff
@@ -23,7 +23,7 @@ import { string_literal } from '../utils/stringify';
 import { Literal } from 'estree';
 import compiler_warnings from '../compiler_warnings';
 import compiler_errors from '../compiler_errors';
-import { ARIARoleDefintionKey, roles, aria, ARIAPropertyDefinition, ARIAProperty } from 'aria-query';
+import { ARIARoleDefinitionKey, roles, aria, ARIAPropertyDefinition, ARIAProperty } from 'aria-query';
```

Comment:
This is scary

Reply:
The misspelling `ARIARoleDefintionKey` is the actual name of the exposed type https://unpkg.com/@types/aria-query@5.0.0/index.d.ts

Reply:
Thanks. Opened https://github.com/DefinitelyTyped/DefinitelyTyped/discussions/62906. And dropped from this PR.